client/css/style.css

```diff
@@ -1097,7 +1097,7 @@ background on hover (unless active) */
 	pointer-events: none;
 	opacity: 0;
 	transform: translateY(16px);
-	transition: all 0.2s ease-in-out;
+	transition: transform 0.2s, opacity 0.2s;
```

Comment:
`all` causes a transition on `right` when toggling the user list, so I went explicit here.
Also went with the same transition as we use everywhere else 🤷‍♂️